google/cloud/storage/google_cloud_cpp_storage_grpc.bzl

```diff
@@ -21,7 +21,6 @@ google_cloud_cpp_storage_grpc_hdrs = [
     "async/client.h",
     "async/connection.h",
     "async/idempotency_policy.h",
-    "async/object_requests.h",
```

Comment:
Q. Is this non-breaking because GCS+gRPC is not yet GA?

Reply:
Correct.